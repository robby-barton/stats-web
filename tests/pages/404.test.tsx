import { render } from "@testing-library/react";

import "@testing-library/jest-dom";
import Custom404 from "@pages/404";

describe("Custom404 page", () => {
	it("renders correctly", async () => {
		const { baseElement } = render(<Custom404 />);

		expect(baseElement).toMatchSnapshot();
	});
});
