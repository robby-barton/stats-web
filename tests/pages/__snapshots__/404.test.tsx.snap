// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Custom404 page renders correctly 1`] = `
<body>
  <div>
    <div>
      <div
        class="header"
      >
        <div
          class="headerTitle"
        >
          <a
            href="/"
          >
            Robby's Ranking
          </a>
        </div>
        <div
          class="headerLink"
        >
          <a
            href="/game-count"
          >
            <span
              class="headingMd"
            >
              Game Count
            </span>
          </a>
        </div>
        <div
          class="headerLink"
        >
          <a
            href="/teams"
          >
            <span
              class="headingMd"
            >
              Teams
            </span>
          </a>
        </div>
        <div
          class="headerLink"
        >
          <a
            href="/about"
          >
            <span
              class="headingMd"
            >
              About
            </span>
          </a>
        </div>
        <div
          class="headerToggle"
        >
          <button
            aria-label="Change to light mode"
            class="css-1f6jnp0"
            title="Change to light mode"
            type="button"
          >
            <span
              class="css-194expn"
            />
            <span>
              üåô
            </span>
            <span>
              ‚òÄÔ∏è
            </span>
          </button>
        </div>
      </div>
      <div
        class="container"
      >
        <div
          class="contentWrap"
        >
          <h1>
            404 - Page not found
          </h1>
        </div>
        <div
          class="footer"
        >
          <a
            href="https://github.com/robby-barton/stats-web"
          >
            &lt;/&gt; - Site source
          </a>
          <br />
          <a
            href="https://github.com/robby-barton/stats-go"
          >
            &lt;/&gt; - Ranking source
          </a>
        </div>
      </div>
    </div>
  </div>
</body>
`;
